style.css

```diff
@@ -326,3 +326,259 @@ hr {
+    margin-top: 80px;
+    line-height: 32px;
+    height: 155px;
+    width: 447px;
+  }
+
+  .section2 {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 13%;
+    justify-content: center;
+  }
+
+  .break-line {
+    display: flex;
+  }
+
+  .list-heading {
+    color: #9490ff;
+    display: flex;
+  }
+
+  .work-left {
+    align-items: left;
+  }
+
+  .work-right .work-left {
+    display: flex;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    text-align: center;
+  }
+
+  .middle-section h1 {
+    text-align: start;
+    display: flex;
+    align-items: center;
+    font-family: 'Roboto', sans-serif;
+    font-size: 64px;
+    font-style: normal;
+  }
+
+  .middle-section hr {
+    height: 0;
+    left: 0;
+    right: 0;
+    border: 1px solid #646369;
+    justify-content: center;
+    margin-top: 35px;
+  }
+
+  .desktop-us {
+    display: flex;
+    justify-content: center;
+    color: #9390ff;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    margin-top: 40px;
+  }
+
+  .mobile-us {
+    display: none;
+  }
+
+  .work-section {
+    display: flex;
+    margin: 80px;
+  }
+
+  .work-section1 {
+    display: flex;
+    flex-direction: row-reverse;
+    margin: 80px;
+  }
+
+  .divide {
+    padding-left: 30px;
+  }
+
+  .divide1 {
+    padding-right: 30px;
+  }
+
+  .template-footer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .template-footer p {
+    width: 684px;
+    height: 61px;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    justify-content: center;
+    padding-top: 3%;
+    padding-bottom: 5%;
+    padding-left: 25%;
+    padding-right: 25%;
+  }
+
+  #name {
+    background: #fff;
+    height: 48px;
+    margin: 0 24px;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    padding: 0 24px;
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-size: 21px;
+    color: #172b4d;
+    width: 70%;
+    margin-bottom: 18px;
+  }
+
+  #email {
+    background: #fff;
+    height: 48px;
+    margin: 0 24px;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    padding: 0 24px;
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-size: 21px;
+    color: #172b4d;
+    width: 70%;
+    margin-bottom: 18px;
+  }
+
+  #msg {
+    height: 170px;
+    left: 24px;
+    margin: 0 24px;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    padding: 0 24px;
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-size: 21px;
+    color: #172b4d;
+    width: 70%;
+  }
+
+  .form-section {
+    margin-left: 20%;
+  }
+
+  .form-button {
+    margin: 5% 38% 5% 38%;
+  }
```

Comment:
- [ ] Kindly review that not only the form button has user interactions but all your buttons. For a better user experience. 👇🏽 
- [ ] Please 🙏🏽 correct the name (SEND) and the size of the form button because it is different from the project buttons.

- Your design

![image](https://user-images.githubusercontent.com/58642949/187029860-f7ccc32c-d9fd-4c6c-a107-bfcc5028edf4.png)

![image](https://user-images.githubusercontent.com/58642949/187030061-dcbfafe7-0a8c-497f-8567-09d72fc948ba.png)

- Figma template

![image](https://user-images.githubusercontent.com/58642949/187030009-1ece6a52-bd2c-4663-a538-4599dfe1190d.png)